README.md

```diff
@@ -46,6 +47,81 @@ npx truffle compile
 npm run networks-inject 
 ```
 
+# Deploying a simple market maker scenario to Rinkeby:
+
+The following script does deploy a simple order market maker order and a relevant
+```
+// Get token id of DAI
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa' --network rinkeby
+
+//export the result
+export TOKEN_ID_DAI=[Result from last call]
+
+// Get token id of TrueUSD
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x0000000000085d4780B73119b644AE5ecd22b376' --network rinkeby
+
+//export the result
+export TOKEN_ID_TUSD=[Result from last call]
+
+// Make sure there is enough funds in the exchange for the intended trades
+npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=0 --amount=30 --network rinkeby&& \
+npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=$TOKEN_ID_TUSD --amount=100 --network rinkeby
+
+// Place market maker order in current auction
+npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_DAI --sellToken=$TOKEN_ID_TUSD --minBuy=1000 --maxSell=998 --validFor=20 --network rinkeby
+
+// Place owl token order
+npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_DAI --sellToken=0 --minBuy=1000 --maxSell=1000 --validFor=20 --network rinkeby
+
+```
+
+Then, the market order can be place, after switching to another account:
+```
+//Deposit funds into exchange:
+npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=$TOKEN_ID_DAI --amount=100 --network rinkeby
+
+// Place market order with 1/2 limit-price
+npx truffle exec scripts/stablex/place_order.js --accountId=1 --buyToken=$TOKEN_ID_TUSD --sellToken=$TOKEN_ID_DAI --minBuy=500 --maxSell=1000 --validFor=5 --network rinkeby
+```
+
+Now, the market can be inspected by:
+```
+//view the market status:
+npx truffle exec scripts/stablex/get_auction_elements.js --network rinkeby
+```
+
+And the output should look like this:
+```
+[ { user: '0x740a98f8f4fae0986fb3264fe4aacf94ac1ee96f',
+    sellTokenBalance: 100000000000000000000,
+    buyToken: 7,
+    sellToken: 3,
+    validFrom: 5247563,
+    validUntil: 5247583,
+    priceNumerator: 1e+21,
```

Comment:
Can the script return the `toString` evaluation of these large numbers?

Reply:
When did we get buy token and sell tokens to have IDs 3 and 7? Is this actually the correct information returned when we make this call with these exact orders? I don't see the 500 as the buy amount of any of these orders? They all look like 1e21 to me.

Reply:
IDs are coming from the rinkeby deployment

The 'toString' is something I would like to exchange in another PR